calculator/src/main/java/com/market/banica/calculator/service/ProductServiceImpl.java

```diff
@@ -197,18 +200,10 @@ private void removeDeletedProductFromAllRecipes(String productName) {
         productBase.getDatabase().forEach((key, value) -> value.getIngredients().remove(productName));
     }
 
-    private Product getProductFromDatabase(String productName) {
-        LOGGER.debug("In getProductFromDatabase private method");
-
-        validateProductExists(productName);
-
-        return productBase.getDatabase().get(productName);
-    }
-
     private void writeProductToDatabase(String newProductName, Product newProduct) {
         LOGGER.debug("In writeProductToDatabase private method");
 
-        announceInterestToOrderBookProductBase(newProductName);
+//        announceInterestToOrderBookProductBase(newProductName);
```

Comment:
Probably forgotten to be deleted ?

Reply:
It is a part of the testing scenario, as I am still doing intensive testing, it is to be fixed before the actual merge, as it is only redirecting data from actual instances to test instances, probably tomorrow morning

Reply:
If this is not going into the main branch, please delete it from the PR and leave it locally for you to test

Reply:
Fixed